src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java

```diff
@@ -4,29 +4,27 @@
 
 import com.exasol.adapter.dynamodb.documentpath.DocumentPathExpression;
 import com.exasol.adapter.metadata.DataType;
-import com.exasol.sql.expression.StringLiteral;
-import com.exasol.sql.expression.ValueExpression;
 
 /**
  * Maps a property of a DynamoDB table and all its descendants to a JSON string.
  */
 public final class ToJsonPropertyToColumnMapping extends AbstractPropertyToColumnMapping {
-    private static final long serialVersionUID = 8019967800146460950L;
+    private static final long serialVersionUID = -6383134783719798072L;
     private final int varcharColumnSize;
-    private final OverflowBehaviour overflowBehaviour;
+    private final MappingErrorBehaviour overflowBehaviour;
```

Comment:
Maybe you can also rename variables

Reply:
No this was intended. it is the overflow behaviour. I just decided that i can gentrify the error behaviours.